tests/pkg/testing/testing.go

```diff
@@ -18,6 +18,48 @@ func NewT(t *testing.T) *T {
 	}
 }
 
+type Logger interface {
+	Log(args ...any)
+	Logf(format string, args ...any)
+	Errorf(format string, args ...interface{})
+	FailNow()
```

Comment:
Do we need these two(Errorf and FailNow)? because we don't have additional logic behind them, just calling the underlying methods?

Reply:
It's required by the methods that accept `require.TestingT`, so I'm just passing it through, so that we don't have to pass in the old `t` for those and for the logs the new one.

Reply:
I found the same approach is used already in the "test-compass/tests/pkg/notifications/asserters/formation_assignments_async.go" file in the `json.AssertJSONStringEquality(...)` fixture where we pass the testing `t` object instead of "onceLogger"? Couldn't we use the same thing here?
Also, if I understand the idea correctly is to use this only for logs that should be printed only once? And not as a substitute for the testing object(in the fixtures and other places)? And not avoiding it purposely and passing one object on both places?

Reply:
If we use the regular object and there are logs within the helper methods, they will be logged multiple times, so it's better to use tOnce in those cases. In the `json.AssertJSONStringEquality` I've just forgot to adapt it to use `tOnce`. In the case of `AssertJSONStringEquality` it just does not print anything, so it doesn't matter which one is used.